CustomDeserializers/Readme.md

```diff
@@ -0,0 +1,16 @@
+Contains example user defined deserializers for azure stream analytics.<br/>
+Interfaces for deserializers are defined in [this nuget](https://www.nuget.org/packages/Microsoft.Azure.StreamAnalytics/).
+
+`ExampleDeserializer` : 
+- Contains a simple deserializer that treats each line as a record.
+
+`EventhubCaptureReader<T>` : 
+- Abstract class with logic to read [EventHub capture](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-capture-overview) format.
+- To use `EventhubCaptureReader` inherit and implement `DeserializeEventData<T>`
+
+`EventhubCaptureCustomEvent` :
+- Example implementation of `EventhubCaptureReader`.
+- Assumes eventhub message body is gzip compressed.
+- Uses `ExampleDeserializer` to deserialize message body.
+
+Run `dotnet publish` to create package with all necessary binaries.<br/>
```

Comment:
Can this be more detailed? So that customers can actually try it out e2e? Additional details asking users to:
1. enable capture from EH to blob store. 
2. use blob store as input to stream job.
3. When we have docs for custom deserializers for cloud and how to use it, we can add a link to that here.

Reply:
Step 3 cannot be explained now, this is the reason why I didn't add additional details. I can make one more update with above details when these are ready.